### PR TITLE
Make the name of the data_bag configurable

### PR DESCRIPTION
**Pull request by ericfode**

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@
 license          "Apache 2.0"
 description      "Installs/Configures Sensu"
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          "2.5.0"
+version          "2.5.4"
```
Comment (portertech) [inline metadata.rb]: Could you please revert this line change. The version will be bumped accordingly when a release is cut.

File: recipes/rabbitmq.rb [hunk 2]
```diff
@@ -31,7 +31,7 @@
     recursive true
   end
 
-  ssl = Sensu::Helpers.data_bag_item("ssl")
+  ssl = Sensu::Helpers.data_bag_item("ssl", flase ,node['sensu']['bag_name'])
```
Comment (amdprophet) [inline recipes/rabbitmq.rb]: Should `flase` be false?

Comment (portertech) [inline recipes/rabbitmq.rb]: typo, `flase` -> `false`

File: attributes/default.rb [hunk 3]
```diff
@@ -43,3 +43,6 @@
 default.sensu.api.host = "localhost"
 default.sensu.api.bind = "0.0.0.0"
 default.sensu.api.port = 4567
+
+#data_bag
+default['sensu']['bag_name'] = "sensu"
```
Comment (portertech) [inline attributes/default.rb]: I believe `data_bag_name` would be more appropriate/descriptive.

File: libraries/sensu_helpers.rb [hunk 4]
```diff
@@ -35,16 +35,16 @@ def gem_binary
         end
       end
 
-      def data_bag_item(item, missing_ok=false)
-        raw_hash = Chef::DataBagItem.load("sensu", item)
+      def data_bag_item(item, missing_ok=false, bag_name="sensu")
```
Comment (portertech) [inline libraries/sensu_helpers.rb]: Let's call the var `data_bag_name`.

**Comment by ericfode** [comment]
@portertech bag_name renamed to data_bag_name

File: recipes/default.rb [hunk 5]
```diff
@@ -63,7 +63,7 @@
     mode 0750
   end
 
-  ssl = Sensu::Helpers.data_bag_item("ssl")
+  ssl = Sensu::Helpers.data_bag_item(node['sensu']['ssl_data_bag_key'], false, node['sensu']['data_bag_name'])
```
Comment (portertech) [inline recipes/default.rb]: Didn't see a default attribute for `ssl_data_bag_key`.

Comment (ericfode) [inline recipes/default.rb]: Adding it now

**Comment by ericfode** [comment]
@portertech done adding data_bag_keys

**Comment by ericfode** [comment]
@portertech I think that everything is ready to be reviewed now :)

**Comment by calebhailey** [comment]
Thanks for following-up @ericfode; we'll look at this again today and respond ASAP. :+1:

**Comment by ericfode** [comment]
@calebhailey Just follwing up. Is there anything I can do?

**Comment by portertech** [comment]
Going to merge this, however, a few minor changes will be made by myself and @amdprophet before a release is cut.

**Comment by portertech** [comment]
Changes https://github.com/sensu/sensu-chef/pull/329

**Comment by portertech** [comment]
Released 2.6.0 https://supermarket.chef.io/cookbooks/sensu

**Pull request merged**